neptune/new/handler.py

```diff
@@ -97,6 +174,33 @@ def log(self,
             timestamp: Optional[float] = None,
             wait: bool = False,
             **kwargs) -> None:
+        """Logs the provided value or a collection of values.
+
+        Available for following field types (`Field types docs page`_):
+            * `Integer`
+            * `Float`
+            * `String`
+            * `Collection`
+            * `File`
+
+
+        Args:
+            value:
```

Comment:
-> value: Value or collection of values to be added to the field.

Reply:
+1